### PR TITLE
Support definition of more than one entity per module

### PR DESCRIPTION
**Pull request by lbolla**
When specifying "resources" with the syntax: "module.path:attribute", include
the attribute in the generated manifest's filename. This way
"module.path:another_attribute" won't conflict with it.

### CODE REVIEW

File: tests/fixtures/attrs/reference/kustomization.yamle [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Example configuration for the webserver
```
Comment (diogobaeder) [inline tests/fixtures/attrs/reference/kustomization.yamle]: This file extension is wrong, it should be ending in `.yaml`, not `.yamle`

**Comment by diogobaeder** [comment]
Except for the typo in the filename (where noted), the changes look good. Have you tested them with a real deployment? Since they change the structure of all the generated directories, and would require a rebuild on current projects that use it (which are expected anyway), I'd like to make sure that it doesn't break the library in general.

**Comment by lbolla** [comment]
> Except for the typo in the filename (where noted), the changes look good. Have you tested them with a real deployment? Since they change the structure of all the generated directories, and would require a rebuild on current projects that use it (which are expected anyway), I'd like to make sure that it doesn't break the library in general.

I haven't tested it on a real deploy. I will do that soon and report here.
We can hold off with merging.

**Comment by lbolla** [comment]
Deployments worked. Can we merge?

**Comment by diogobaeder** [comment]
Merged. Waiting for Travis to finish building so that we have a new version released. The version will be 0.5.0, since this change has a bit more impact than a regular bugfix, as it changes the general structure of the generated YAML files.

**Pull request merged**